mmv1/third_party/terraform/services/resourcemanager/ephemeral_google_service_account_id_token_test.go

```diff
@@ -51,6 +52,27 @@ func TestAccEphemeralServiceAccountIdToken_withDelegates(t *testing.T) {
 	})
 }
 
+func TestAccEphemeralServiceAccountIdToken_withEmptyDelegates(t *testing.T) {
```

Comment:
This comment is relevant to how we handle empty delegates: https://github.com/GoogleCloudPlatform/magic-modules/pull/12448/files#r1861191127